portfolio/src/main/webapp/index.html

```diff
@@ -8,8 +8,12 @@
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
+      <h1>David Ihim's Portfolio</h1>
+        <p>
```

Comment:
nit: this < p > span should be aligned with the previous < h 1 > to indicate that they are children at the same depth of the DOM tree.

Reply:
Got it, thanks. Looks like I aligned it in a later commit